src/System.Security.Claims/src/System/Security/Claims/Claim.cs

```diff
@@ -277,6 +359,10 @@ public string OriginalIssuer
         {
             get
             {
+                if (_properties == null)
+                {
+                    _properties = new Dictionary<string, string>();
+                }
```

Comment:
The change is to avoid the allocation of another object per Claim per request when the Properties dictionary is often not used. The lock object, while smaller, would still have that allocation.